sycl/doc/PreprocessorMacros.md

```diff
@@ -33,6 +33,10 @@ SYCL 2020.
 
 Disables all deprecation warnings in SYCL runtime headers, including SYCL 1.2.1 deprecations.
 
+### `SYCL_DISABLE_FALLBACK_ASSERT`
+
+Forces fallback assert feature implementation disable
```

Comment:
Could you put a more details on expected default behavior and how setting environment variable changes it?

Reply:
Added